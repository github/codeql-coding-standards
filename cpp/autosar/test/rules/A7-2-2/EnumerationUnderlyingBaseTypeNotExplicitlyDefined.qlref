rules/A7-2-2/EnumerationUnderlyingBaseTypeNotExplicitlyDefined.ql