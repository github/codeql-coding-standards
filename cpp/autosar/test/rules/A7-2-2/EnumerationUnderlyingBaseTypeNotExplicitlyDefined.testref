cpp/common/test/rules/enumerationnotdefinedwithanexplicitunderlyingtype/EnumerationNotDefinedWithAnExplicitUnderlyingType.ql