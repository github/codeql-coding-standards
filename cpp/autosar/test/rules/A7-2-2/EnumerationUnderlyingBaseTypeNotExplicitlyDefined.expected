| test.cpp:1:6:1:6 | C | Base type of enumeration is not explicitly specified. |
| test.cpp:5:12:5:12 | A | Base type of enumeration is not explicitly specified. |
